packages/amplify-provider-awscloudformation/src/aws-utils/aws-s3.ts

```diff
@@ -155,11 +155,30 @@ export class S3 {
       },
       (response?) => response.Versions?.map(({ Key, VersionId }) => ({ Key, VersionId })),
       async (response?) =>
-        response && response.IsTruncated ? { KeyMarker: response.NextKeyMarker, VersionIdMarker: response.NextVersionIdMarker } : undefined,
+        response && response.IsTruncated
+          ? { KeyMarker: response.NextKeyMarker, VersionIdMarker: response.NextVersionIdMarker, Prefix: response.Prefix }
+          : undefined,
     );
     return result;
   }
 
+  public async deleteDirectory(bucketName: string, dirPath: string): Promise<void> {
+    logger('deleteDirectory.s3.getAllObjectVersions', [{ BucketName: bucketName }])();
+    const allObjects = await this.getAllObjectVersions(bucketName, { Prefix: dirPath });
+    const chunkedResult = _.chunk(allObjects, 1000);
+    for (let chunk of chunkedResult) {
+      logger('deleteDirectory.s3.deleteObjects', [{ Bucket: bucketName }])();
```

Comment:
We should add a `x of y` based on the number of chunks otherwise we are missing information here.

Reply:
Are you referring to adding this for the logger?

Reply:
Yes, we could enrich the logging with it:
`deleteDirectory.s3.deleteObjects (1 of 3)`
`deleteDirectory.s3.deleteObjects (2 of 3)`
`deleteDirectory.s3.deleteObjects (3 of 3)`

progress instead of seeing 3 entries of `deleteDirectory.s3.deleteObjects` only.